puente-etl/utils/constants.py

```diff
@@ -0,0 +1,104 @@
+class ColumnOrder:
+    # Enforce column order for relational table outputs
+    FORM_SPECIFICATIONS = [
+        'custom_form_id',
+        'custom_form_organizations',
+        'custom_form_class',
+        'custom_form_type_of_form',
+        'custom_form_name',
+        'custom_form_description',
+        'custom_form_header',
+        #
+        # Excluding these for now for readability:
+        #
+        # 'custom_form_updated_at',
+        # 'custom_form_created_at',
```

Comment:
Thanks! We'll definitely need

- `custom_form_created_at`
- `custom_form_active`

Reply:
Will add these back